app/build.gradle

```diff
@@ -108,6 +108,8 @@ buildscript {
 }
 
 dependencies {
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
You can move id version to a separate 'module'. If you see in line 92 of this file, there are lots of 'extensions' created. You can do it too